src/status_im/pairing/core.cljs

```diff
@@ -326,15 +326,15 @@
              contacts))
 
 (defn handle-sync-installation
-  [{:keys [db] :as cofx} {:keys [contacts multiaccount chat]} sender]
+  [{:keys [db] :as cofx} {:keys [contacts account chat]} sender]
```

Comment:
is it because status-go is calling account? I think this is slowly changing though because we need to clearly distinguish accounts and multiaccounts

Reply:
no, it's the `defrecord` that uses `account`, we can change that I guess, but best to do in a separate PR